x-pack/plugins/security_solution/public/entity_analytics/components/entity_details_flyout/tabs/risk_inputs.tsx

```diff
@@ -30,6 +39,73 @@ export interface AlertRawData {
   _id: string;
 }
 
+const FieldLabel = styled.span`
+  font-weight: ${euiLightVars.euiFontWeightMedium};
+  color: ${euiLightVars.euiTitleColor};
+`;
+
+const ContextsTable: React.FC<{ riskScore?: UserRiskScore | HostRiskScore; loading: boolean }> = ({
+  riskScore,
+  loading,
+}) => {
+  const criticalityLevel = useMemo(() => {
+    if (!riskScore) {
+      return undefined;
+    }
+
+    if (isUserRiskScore(riskScore)) {
+      return riskScore.user.risk.criticality_level;
+    }
+
+    return riskScore.host.risk.criticality_level;
+  }, [riskScore]);
+
+  const columns = [
+    {
+      name: (
+        <FormattedMessage
+          id="xpack.securitySolution.flyout.entityDetails.fieldColumnTitle"
+          defaultMessage="Field"
+        />
+      ),
+      field: 'label',
+      render: (label: string) => <FieldLabel>{label}</FieldLabel>,
+    },
+    {
+      name: (
+        <FormattedMessage
+          id="xpack.securitySolution.flyout.entityDetails.valuesColumnTitle"
+          defaultMessage="Values"
+        />
+      ),
+      field: 'field',
+      render: (field: string | undefined, { render }: { render: () => JSX.Element }) => render(),
+    },
+  ];
+
+  const items = [
```

Comment:
nit: Add `useMemo` to avoid unnecessary rerenders.

Reply:
Done in commit [memoize](https://github.com/elastic/kibana/pull/174638/commits/ca706c97516643053a53da1ca336e4bfe78fcb2e)